org.kde.falkon.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "org.kde.falkon",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "falkon",
+    "rename-icon": "falkon",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--filesystem=home",
```

Comment:
what does this need in the whole home directory that it can't get with portals? given the threat model it's always good to scope down a web browser as much as we can.

Reply:
My bad; I seem to forget the existence of portals. Thanks, updated.